### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by scoute-dich**

### CODE REVIEW

**Comment by loperahmed** [comment]
Hi Gaukler, I designed a great logo Icon for this beautiful android Foss browser, If you like it you are welcome, I will send you all requested type files, SVG, PNG etc. also no problem if you want something to design them for you related to the FOSS Browser for free ;)


FossBrowser

File: app/src/main/res/values-ar/strings.xml [hunk 1]
```diff
@@ -41,18 +41,18 @@
     <string name="menu_filter">فرز بواسطة…</string>
     <string name="menu_share_link">مشاركة الرابط</string>
     <string name="menu_save_home">حفظ في الصفحة الرئيسية</string>
-    <string name="menu_save_bookmark">احفظ كعلامة</string>
+    <string name="menu_save_bookmark">حفظ كإشارة مرجعية</string>
     <string name="menu_save_as">حفظ كـ …</string>
-    <string name="menu_save_pdf">إحفظ ك PDF</string>
```
Comment (loperahmed) [inline app/src/main/res/values-ar/strings.xml:46]: حفظ ك PDF

File: app/src/main/res/values-ar/strings.xml [hunk 2]
```diff
@@ -41,18 +41,18 @@
...
-    <string name="menu_shareClipboard">انسخ إلى الحافظة</string>
-    <string name="menu_other_searchSite">ابحث في الموقع</string>
```
Comment (loperahmed) [inline app/src/main/res/values-ar/strings.xml:48]: بحث في الموقع

File: app/src/main/res/values-ar/strings.xml [hunk 3]
```diff
@@ -41,18 +41,18 @@
...
+    <string name="menu_save_pdf">حفظ كـ PDF</string>
+    <string name="menu_shareClipboard">نسخ للحافظة</string>
+    <string name="menu_other_searchSite">البحث في الموقع</string>
     <string name="menu_other_info">حول \"FOSS Browser\"</string>
     <!-- Dialog -->
     <string name="dialog_title_hint">العنوان:</string>
     <string name="dialog_extension_hint">الامتداد (مثلا: \"‎.jpg\"):</string>
     <string name="dialog_sign_in_username">اسم المستخدم</string>
     <string name="dialog_sign_in_password">كلمة السر</string>
-    <string name="dialog_title_download">تريد تنزيل الملف؟</string>
```
Comment (loperahmed) [inline app/src/main/res/values-ar/strings.xml:55]: تنزيل الملف؟

**Review by loperahmed — APPROVED** [review]

**Comment by scoute-dich** [comment]
> Hi Gaukler, I designed a great logo Icon for this beautiful android Foss browser, If you like it you are welcome, I will send you all requested type files, SVG, PNG etc. also no problem if you want something to design them for you related to the FOSS Browser for free ;)
> 
> FossBrowser

Yes, I'm interested. It looks great. I'll have more time next week. So let us talk then.

**Comment by loperahmed** [comment]
Hi Gaukler,
Hope you are well, here is the icons for **Foss browser**, if you have any requests or questions about it you are welcome, just shut me a message here or on telegram by this **username** _@MrDemha_
[Documents.zip](https://github.com/scoute-dich/browser/files/6526140/Documents.zip)

**Comment by loperahmed** [comment]
here is the other drawable files:
[Drawable-files.zip](https://github.com/scoute-dich/browser/files/6526259/Drawable-files.zip)

**Comment by scoute-dich** [comment]
I just forgot your icons. Thanks a lot. At the moment I'm testing the icons. I must get used to it. Following questions, if you still want to help:

- Can you try to use the svg-format instead of psd?
- Are you willing to make some banners for the PlayStore and F-Droid? Then I'll give you the specifications.

**Comment by loperahmed** [comment]
Hi Gaukler,
Hope you doing well, here is your requested files(SVG). For you asking about banners, you welcome, tell me witch banner you want for G-play & f-droid, just tell me an example that you imagine and I will try it.

[SVGsRequested.zip](https://github.com/scoute-dich/browser/files/6642902/SVGsRequested.zip)

**Pull request merged**